### PR TITLE
Enable office user to edit existing approved sit request.

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Description

Enables an office user to edit an existing SIT request that's been approved. This allows them to change the authorized start date and the authorization note associated with the request.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make server_run
make office_client_run
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @ntwyman
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/164252054) for this change

## Screenshots

May-10-2019 13-26-32

### CODE REVIEW

File: cypress/integration/office/storageInTransitPanel.js [hunk 1]
```diff
@@ -198,4 +198,18 @@ function officeUserApprovesSITRequest() {
...
+  cy.get('textarea[name="authorization_notes"]').type('this is also a note', { force: true, delay: 150 });
+
+  cy
+    .get('[data-cy="sit-editor-save-button"]')
+    .contains('Save')
+    .click();
+
+  cy.patientReload();
+
+  cy.get('[data-cy="sit-authorization-notes"]').contains('this is also a note');
```
Comment (reggieriser) [inline cypress/integration/office/storageInTransitPanel.js]: Should we also check that the authorized_start_date was successfully changed?

**Comment by Ryan-Koch** [comment]
> Generally looks good and seemed to work fine when I tested. I did notice, though, that the edit form shows a SIT number of 9999999999 when there shouldn't be a SIT number at this point. Should we change it to a "n/a" or something? Also added one Cypress test note.

Nice catch on the SIT number. It looks like that was a placeholder for the value field that had been put in at some point along the way. I'll change that to n/a and push the change.

File: cypress/integration/office/storageInTransitPanel.js (hunk 1 above)
Comment (Ryan-Koch) [reply cypress/integration/office/storageInTransitPanel.js]: Yeah that makes sense. I'll add that in.

Comment (Ryan-Koch) [reply cypress/integration/office/storageInTransitPanel.js]: Pushed.

**Pull request merged**